cryptokit/Sources/CryptoKitSrc/hash.swift

```diff
@@ -68,3 +67,299 @@ public func SHA512(
     let hashData = hash.withUnsafeBytes { Data($0) }
     hashData.copyBytes(to: outputPointer, count: hashData.count)
 }
+
+@_cdecl("NewMD5")
+public func NewMD5() -> UnsafeMutableRawPointer {
+    let hasher = UnsafeMutablePointer<Insecure.MD5>.allocate(capacity: 1)
+    hasher.initialize(to: Insecure.MD5())
+    return UnsafeMutableRawPointer(hasher)
+}
+
+@_cdecl("MD5Write")
+public func MD5Write(_ ptr: UnsafeMutableRawPointer, _ data: UnsafePointer<UInt8>, _ length: Int) -> Int {
+    let hasher = ptr.assumingMemoryBound(to: Insecure.MD5.self)
+    let buffer = UnsafeRawBufferPointer(start: data, count: length)
+
+    hasher.pointee.update(data: buffer)
+    
+    return length // Always return full length as Swift doesn't have partial writes
```

Comment:
If CryptoKit always do full writes, then there is no need to return the bytes written. Change all the `*Write` function signature to return nothing.